build.gradle

```diff
@@ -729,6 +729,17 @@ subprojects {
 
             if (project.name.is('checker')) {
                 tasks += ['nullnessExtraTests', 'commandLineTests', 'tutorialTests']
+            } else {
+                tasks += ['test']
```

Comment:
For parallelism/uniformity, why not have a nonJunitTests and a 'test' target for each project?  Then they would all be organized in the same way.  It doesn't slow things down much, if any, because it won't require more CI jobs.
Maybe there is a reason not to do that, but I don't see it offhand.